scripts/use-canary-version.js

```diff
@@ -60,7 +60,7 @@ function fetchPackageInfo(pkg) {
   const manifest = JSON.parse(data);
 
   const canaryDependencies = {
-    react: peerDependencies['react'],
+    react: peerDependencies.react,
```

Comment:
Just curious on why it's needed to access 'react' differently than 'react-native'?

Reply:
Unclear! This change was made automatically.